store/index.js

```diff
@@ -1,10 +1,5 @@
 /* eslint import/no-extraneous-dependencies : 0 */
-
-import Vuex from 'vuex'
-
-/* Modules */
-
-const createStore = () => new Vuex.Store({
+export default {
```

Comment:
Pretty sure you need the Vuex.Store to let Vue know this is vuex.

Reply:
Hey man, just learned that nuxt deprecated this way (classic mode) of creating a store instance. So by default it will recognize the files in the store folder as modules with the index as the root.

Reply:
@troitroi Can you send me the link about that ?

Reply:
hey man here it is https://nuxtjs.org/guide/vuex-store/